src/probe/Kconfig

```diff
@@ -5,11 +5,14 @@
 menu "Probe"
 
 config PROBE
-	bool "Probes enabled"
+	tristate "Probes enabled"
+	default m if LIBRARY_DEFAULT_MODULAR
 	default y if CAVS
 	help
 	  Select for enabling debug probes to extract/inject buffers
 
+if PROBE != n
+
 config PROBE_POINTS_MAX
 	int "Maximum probe points"
 	depends on PROBE
```

Comment:
I think now with the `if` above you don't need these `depends` any more - here and below?